src/Microsoft.DotNet.ProjectModel/Resolution/PackageDependencyProvider.cs

```diff
@@ -113,6 +113,11 @@ private static IEnumerable<string> GetDependencies(string path)
         {
             using (var peReader = new PEReader(File.OpenRead(path)))
             {
+                if (!peReader.HasMetadata)
+                {
+                    throw new InvalidDataException($"{path} does not contain managed metadata");
```

Comment:
Shouldn't there be at least a warning? If it's silently skipped then the app will most likely crash at runtime with weird messages with the user having no idea how to approach debugging.

---
In reply to: [61773268](https://github.com/dotnet/cli/pull/2814#discussion_r61773268) [](ancestors = 61773268)